Example Apps/Example Swift/ProxyManager.swift

```diff
@@ -74,26 +78,33 @@ private extension ProxyManager {
     /// - Returns: A SDLConfiguration object
     class var iapConfiguration: SDLConfiguration {
         let lifecycleConfiguration = SDLLifecycleConfiguration(appName: ExampleAppName, fullAppId: ExampleFullAppId)
-        return setupManagerConfiguration(with: lifecycleConfiguration)
+        return setupManagerConfiguration(with: lifecycleConfiguration, enableRemote: false)
     }
 
     /// Configures a TCP transport layer with the IP address and port of the remote SDL Core instance.
     ///
     /// - Returns: A SDLConfiguration object
     class var tcpConfiguration: SDLConfiguration {
         let lifecycleConfiguration = SDLLifecycleConfiguration(appName: ExampleAppName, fullAppId: ExampleFullAppId, ipAddress: AppUserDefaults.shared.ipAddress!, port: UInt16(AppUserDefaults.shared.port!)!)
-        return setupManagerConfiguration(with: lifecycleConfiguration)
+        return setupManagerConfiguration(with: lifecycleConfiguration, enableRemote: true)
     }
 
     /// Helper method for setting additional configuration parameters for both TCP and iAP transport layers.
     ///
     /// - Parameter lifecycleConfiguration: The transport layer configuration
     /// - Returns: A SDLConfiguration object
-    class func setupManagerConfiguration(with lifecycleConfiguration: SDLLifecycleConfiguration) -> SDLConfiguration {
+    class func setupManagerConfiguration(with lifecycleConfiguration: SDLLifecycleConfiguration, enableRemote: Bool) -> SDLConfiguration {
         lifecycleConfiguration.shortAppName = ExampleAppNameShort
         let appIcon = UIImage(named: ExampleAppLogoName)?.withRenderingMode(.alwaysOriginal)
         lifecycleConfiguration.appIcon = appIcon != nil ? SDLArtwork(image: appIcon!, persistent: true, as: .PNG) : nil
         lifecycleConfiguration.appType = .default
+
+        // On actional hardware, the app requires permissions to do remote control which this example app will not have.
+        // Only use the remote control type on the TCP connection.
+        if (enableRemote) {
```

Comment:
```suggestion
        if enableRemote {
```

Don't use `(` in Swift `if`s or `guard`s unless there are inner comparisons, e.g.

```swift
if (test1 == something) && ((test2 == somethingElse) || (foo = bar))
```

Even though we use parenthesis to make clearer what is being compared, we still don't use "outer" ones for the whole check.